### PR TITLE
[zh-cn] resync: docs/contribute/style/style-guide.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 1]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+### Clarification about "resource"
+
+Kubernetes uses the word "resource" to refer to API resources, such as `pod`, `deployment`, and so on. We also use "resource" to talk about CPU and memory requests and limits. Always refer to API resources as "API resources" to avoid confusion with CPU and memory resources.
+-->
+## 引用 Kubernetes API 资源
+
+本节讨论我们如何在文档中引用 API 资源。
+
+### 澄清“资源”
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:397]:
```suggestion
### 有关 “资源” 的阐述
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 2]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+Kubernetes 使用“resource”一词来指代 API 资源，例如 `pod`、`deployment` 等。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:399]:
```suggestion
Kubernetes 使用 “resource” 一词来指代 API 资源，例如 `pod`、`deployment` 等。 
```
If possible, give a space around the quotation mark `“”`.

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 3]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+-->
+## 引用 Kubernetes API 资源
+
+本节讨论我们如何在文档中引用 API 资源。
+
+### 澄清“资源”
+
+Kubernetes 使用“resource”一词来指代 API 资源，例如 `pod`、`deployment` 等。 
+我们还使用“resource”来谈论 CPU 和内存请求和限制。 
+所以始终将 API 资源称为“API resources”以避免与 CPU 和内存资源混淆。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:401]:
```suggestion
所以始终将 API 资源称为 “API resources” 以避免与 CPU 和内存资源混淆。
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 4]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+Always use "resource" or "object" when referring to an API resource in docs. For example, use "a `Secret` object" over just "a `Secret`".
+-->
+### 何时使用 Kubernetes API 术语
+
+不同的 Kubernetes API 术语是：
+
+- 资源类型：API URL 中使用的名称（如 `pods`、`namespaces`）
+- 资源：资源类型的单个实例（如 `pod`、`secret`）
+- 对象：作为“目标记录”的资源。对象是集群特定部分的期望状态， 
+Kubernetes 控制平面试图维护该状态。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:421]:
```suggestion
  Kubernetes 控制平面尝试维护该状态。
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 5]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+- Resource type: the name used in the API URL (such as `pods`, `namespaces`)
+- Resource: a single instance of a resource type (such as `pod`, `secret`)
+- Object: a resource that serves as a "record of intent". An object is a desired state for a specific part of your cluster, which the Kubernetes control plane tries to maintain.
+
+Always use "resource" or "object" when referring to an API resource in docs. For example, use "a `Secret` object" over just "a `Secret`".
+-->
+### 何时使用 Kubernetes API 术语
+
+不同的 Kubernetes API 术语是：
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:416]:
```suggestion
不同 Kubernetes API 术语的说明如下：
```
only suggestion. 这句与上下文有所割裂，没连起来。

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 6]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+Don't split an API object name into separate words. For example, use `PodTemplateList`, not Pod Template List.
+
+For more information about PascalCase and code formatting, please review the related guidance on [Use upper camel case for API objects](/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) and [Use code style for inline code, commands, and API objects](/docs/contribute/style/style-guide/#code-style-inline-code).
+
+For more information about Kubernetes API terminologies, please review the related guidance on [Kubernetes API terminology](/docs/reference/using-api/api-concepts/#standard-api-terminology).
+-->
+### API 资源名称
+
+始终使用 [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case)
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:441]:
```suggestion
始终使用 [UpperCamelCase](https://zh.wikipedia.org/wiki/%E9%A7%9D%E5%B3%B0%E5%BC%8F%E5%A4%A7%E5%B0%8F%E5%AF%AB) 
```

**Review by windsonsea — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/contribute/style/style-guide.md (hunk 2 above)
Comment (hs0210) [reply content/zh-cn/docs/contribute/style/style-guide.md]: Thanks, done. I have added space around all the quotation mark `“”`.

File: content/zh-cn/docs/contribute/style/style-guide.md (hunk 3 above)
Comment (hs0210) [reply content/zh-cn/docs/contribute/style/style-guide.md]: Done.

File: content/zh-cn/docs/contribute/style/style-guide.md (hunk 4 above)
Comment (hs0210) [reply content/zh-cn/docs/contribute/style/style-guide.md]: Done.

File: content/zh-cn/docs/contribute/style/style-guide.md (hunk 5 above)
Comment (hs0210) [reply content/zh-cn/docs/contribute/style/style-guide.md]: Thanks, done.

File: content/zh-cn/docs/contribute/style/style-guide.md (hunk 6 above)
Comment (hs0210) [reply content/zh-cn/docs/contribute/style/style-guide.md]: Done.

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 7]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+不要将 API 对象的名称切分成多个单词。例如，使用 `PodTemplateList`， 
+不要使用 Pod Template List。
+
+有关 PascalCase 和代码格式的更多信息， 
+请查看[对 API 对象使用大写驼峰式命名法](/zh-cn/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) 
+和[为行间代码、命令与 API 对象使用代码样式](/zh-cn/docs/contribute/style/style-guide/#code-style-inline-code)。
+
+有关 Kubernetes API 术语的更多信息， 
+请查看[Kubernetes API 术语](/zh-cn/docs/reference/using-api/api-concepts/#standard-api-terminology)的相关指南。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/style/style-guide.md:455]:
```suggestion
请查看 [Kubernetes API 术语](/zh-cn/docs/reference/using-api/api-concepts/#standard-api-terminology)的相关指南。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
You need not to reply each comment. If you accepted the comment and performed the change, click the `Resolve conversation` button. Then all reviewers will know you have changed.

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 8]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+Always use "resource" or "object" when referring to an API resource in docs. For example, use "a `Secret` object" over just "a `Secret`".
+-->
+### 何时使用 Kubernetes API 术语
+
+不同 Kubernetes API 术语的说明如下：
+
+- 资源类型：API URL 中使用的名称（如 `pods`、`namespaces`）
+- 资源：资源类型的单个实例（如 `pod`、`secret`）
+- 对象：作为 “目标记录” 的资源。对象是集群特定部分的期望状态，
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/style/style-guide.md:420]:
```suggestion
- 对象：作为 “意向记录” 的资源。对象是集群特定部分的期望状态， 
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 9]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+Don't split an API object name into separate words. For example, use `PodTemplateList`, not Pod Template List.
+
+For more information about PascalCase and code formatting, please review the related guidance on [Use upper camel case for API objects](/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) and [Use code style for inline code, commands, and API objects](/docs/contribute/style/style-guide/#code-style-inline-code).
+
+For more information about Kubernetes API terminologies, please review the related guidance on [Kubernetes API terminology](/docs/reference/using-api/api-concepts/#standard-api-terminology).
+-->
+### API 资源名称
+
+始终使用 [UpperCamelCase](https://zh.wikipedia.org/wiki/%E9%A7%9D%E5%B3%B0%E5%BC%8F%E5%A4%A7%E5%B0%8F%E5%AF%AB) 
+（也称为 PascalCase）和代码格式化来格式化 API 资源名称。
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/style/style-guide.md:442]:
```suggestion
（也称为 PascalCase）和代码格式来表达 API 资源名称。
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 10]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+对于 HTML 文档中的内联代码，请使用 `<code>` 标记。 
+在 Markdown 文档中，使用反引号 (`` ` ``)。
+
+不要将 API 对象的名称切分成多个单词。例如，使用 `PodTemplateList`， 
+不要使用 Pod Template List。
+
+有关 PascalCase 和代码格式的更多信息， 
+请查看[对 API 对象使用大写驼峰式命名法](/zh-cn/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) 
+和[为行间代码、命令与 API 对象使用代码样式](/zh-cn/docs/contribute/style/style-guide/#code-style-inline-code)。
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/style/style-guide.md:452]:
```suggestion
和[针对内嵌代码、命令与 API 对象使用代码样式](/zh-cn/docs/contribute/style/style-guide/#code-style-inline-code)。
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 11]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+Always use "resource" or "object" when referring to an API resource in docs. For example, use "a `Secret` object" over just "a `Secret`".
+-->
+### 何时使用 Kubernetes API 术语
+
+不同 Kubernetes API 术语的说明如下：
+
+- 资源类型：API URL 中使用的名称（如 `pods`、`namespaces`）
+- 资源：资源类型的单个实例（如 `pod`、`secret`）
+- 对象：作为 “意向记录  ” 的资源。对象是集群特定部分的期望状态， 
+  Kubernetes 控制平面试图维护该状态。
```
Comment (Sea-n) [inline content/zh-cn/docs/contribute/style/style-guide.md:421]: Not sure if it's correct.

```suggestion
  该状态由 Kubernetes 控制平面負責维护。
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 12]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+Don't split an API object name into separate words. For example, use `PodTemplateList`, not Pod Template List.
+
+For more information about PascalCase and code formatting, please review the related guidance on [Use upper camel case for API objects](/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) and [Use code style for inline code, commands, and API objects](/docs/contribute/style/style-guide/#code-style-inline-code).
+
+For more information about Kubernetes API terminologies, please review the related guidance on [Kubernetes API terminology](/docs/reference/using-api/api-concepts/#standard-api-terminology).
+-->
+### API 资源名称
+
+始终使用 [UpperCamelCase](https://zh.wikipedia.org/wiki/%E9%A7%9D%E5%B3%B0%E5%BC%8F%E5%A4%A7%E5%B0%8F%E5%AF%AB) 
+（也称为 PascalCase）和代码格式来表达 API 资源名称。
```
Comment (Sea-n) [inline content/zh-cn/docs/contribute/style/style-guide.md:442]: The English word "UpperCamelCase" is not a proper noun, just "upper camel case" written in pascal case.

```suggestion
始终使用[大寫駝峰式命名法](https://zh.wikipedia.org/wiki/%E9%A7%9D%E5%B3%B0%E5%BC%8F%E5%A4%A7%E5%B0%8F%E5%AF%AB)
（PascalCase）和代码格式来表达 API 资源名称。
```

File: content/zh-cn/docs/contribute/style/style-guide.md [hunk 13]
```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
...
+
+对于 HTML 文档中的内联代码，请使用 `<code>` 标记。 
+在 Markdown 文档中，使用反引号 (`` ` ``)。
+
+不要将 API 对象的名称切分成多个单词。例如，使用 `PodTemplateList`， 
+不要使用 Pod Template List。
```
Comment (Sea-n) [inline content/zh-cn/docs/contribute/style/style-guide.md:448]:
```suggestion
不要将 API 对象的名称切分成多个单词。例如請使用 `PodTemplateList` 而非 `Pod Template List`。
```

File: content/zh-cn/docs/contribute/style/style-guide.md (hunk 11 above)
Comment (hs0210) [reply content/zh-cn/docs/contribute/style/style-guide.md]: It's OK, the original translation is more blunt.

**Pull request merged**